README.md

```diff
@@ -50,13 +50,33 @@ git clone https://github.com/Start9Labs/documentation.git && cd documentation
 git submodule update --init
 ```
 
+1. Prep NVM:
+
+```
+nvm use v16.11 || nvm install v16.11 && nvm use v16.11
+```
+
+1. Ensure poetry is updated
+
+```
+poetry update
+```
+
 1. Run setup script to install project dependencies:
 
 ```
 chmod a+x site/_utils/setup.sh
 ./site/_utils/setup.sh
 ```
 
+1. Make sure sphinx packages in the package.lock file are preset, for consistency:
+
+```
+cd sphinx-scylladb-theme
+npm ci
```

Comment:
`ci` or `i` ?

Reply:
Lucy says:
npm ci will just make sure that the packages in the package.lock file are present, which is best for consistency. package.lock would get updated if I (or someone) made changes to the submodule that required new packages/package versions